src/app.js

```diff
@@ -1,21 +1,30 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const corectUrl = new URL(req.url, `http://${req.headers.host}`);
+
+  if (!corectUrl.pathname.startsWith('/file')) {
+    res.statusCode = 404;
+
+    res.end('The pathname should start with "/file"');
+  }
+
+  const fileName = req.url.slice(6) || 'index.html';
+
+  try {
+    const data = fs.readFileSync(`./public/${fileName}`, 'utf-8');
```

Comment:
Using 'readFileSync' in the server will block other requests as it waits for the file to be read completely. It's better to use 'readFile' instead and handle the response in a callback function, to allow for non-blocking operation.